src/service/interface.ts

```diff
@@ -96,7 +97,7 @@ export interface ITrips_v2 {
   ): Promise<Result<Trips.TripsQuery, APIError>>;
   getSingleTrip(
     query: Trips.TripsQueryWithJourneyIds
-  ): Promise<Result<Trips.TripPattern, APIError>>;
+  ): Promise<Result<Trips.TripPattern, Boom>>;
```

Comment:
Updating the return type here seems to better support multiple possible errors. I can return `Boom.notFound()` or `Boom.internal()` without relying on the [limited APIError class](https://github.com/AtB-AS/atb-bff/blob/main/src/service/types.ts#L312-L333). This seems like a good solution to me, but it's not done anywhere else in interface.ts, so I might be missing something.